### DISCUSSION TITLE
问一个小白的问题…… 请问为什么设置颜色样式和我输入的不一样？

### DISCUSSION THREAD

**Discussion by YummyCocoa** [Mintimate/oh-my-rime#2] [Q&A] [answered]
请问，在样式设置里，设置的样式和实际编译展示的不一样？

比如，我设置的颜色是`#E74435`：
<img width="714" alt="image" src="https://user-images.githubusercontent.com/132128788/235291413-8862faa5-f9e8-43da-b5d5-1fa8ee869445.png">

但是实际上是蓝色`#3140E4`？

<img width="648" alt="image" src="https://user-images.githubusercontent.com/132128788/235291442-6cbae160-916a-4260-ab67-677e22be43b7.png">

**Comment by Mintimate** [comment, marked as answer]
哈哈，这个问题其实很简单。

`0x`开头的颜色格式和`#`开头，虽然都是16进制颜色编码。但是，`0x`和#开头的16进制比，如果颜色为#xxyyzz，那么转为0x开头就是xx和zz互换：0xzzyyxx。

其实对应博客文档也有，对更多配置感兴趣，可以参考rime官方文档的同时，看看博客图文：[Mintimate's Blog](https://www.mintimate.cn/2023/03/18/rimeQuickInit/#%E9%85%8D%E8%89%B2%E6%A0%B7%E5%BC%8F)。

所以，`#E74435`转`0x`应该是: `0x3544E7`：
<img width="549" alt="image" src="https://user-images.githubusercontent.com/43734212/235291664-c26d7508-e404-4530-9464-1a337f15c3be.png">
